### PR TITLE
fix(motion): vim-matchup - matchup_matchparen_offscreen method='popup…

### PR DESCRIPTION
**Pull request by jay-babu**
…' #279

### CODE REVIEW

**Review by mehalter — CHANGES_REQUESTED** [review]
Awesome, do you think we should also add

```lua
vim.g.matchup_matchparen_deferred = 1
```

to the default options as well?

File: lua/astrocommunity/motion/vim-matchup/init.lua [hunk 1]
```diff
@@ -1,5 +1,6 @@
 return {
   "nvim-treesitter/nvim-treesitter",
   dependencies = { "andymass/vim-matchup" },
+  init = function() vim.g.matchup_matchparen_offscreen = { method = "popup" } end,
```
Comment (jay-babu) [inline lua/astrocommunity/motion/vim-matchup/init.lua:4]:
```suggestion
  init = function() 
    vim.g.matchup_matchparen_offscreen = { method = "popup" }
    vim.g.matchup_matchparen_deferred = 1
  end,
```

**Comment by jay-babu** [comment]
> Awesome, do you think we should also add
> 
> ```lua
> vim.g.matchup_matchparen_deferred = 1
> ```
> 
> to the default options as well?

sure, why not. i just turned the issue into a PR

**Review by mehalter — APPROVED** [review]
LGTM!

**Pull request merged**